docs/sources/datasources/azuremonitor.md

```diff
@@ -10,49 +10,38 @@ weight = 300
 
 The Azure Monitor data source supports multiple services in the Azure cloud:
 
-- **[Azure Monitor Metrics]({{< relref "#querying-the-azure-monitor-service" >}})** (or Metrics) is the platform service that provides a single source for monitoring Azure resources.
-- **[Azure Monitor Logs]({{< relref "#querying-the-azure-log-analytics-service" >}})** (or Logs) gives you access to log data collected by Azure Monitor.
-- **[Application Insights]({{< relref "#querying-the-application-insights-service" >}})** is an extensible Application Performance Management (APM) service for web developers on multiple platforms and can be used to monitor your live web application - it will automatically detect performance anomalies.
-- **[Application Insights Analytics]({{< relref "#query-the-application-insights-analytics-service" >}})** allows you to query [Application Insights data](https://docs.microsoft.com/en-us/azure/azure-monitor/app/analytics) using the same query language used for Azure Log Analytics.
+- **[Azure Monitor Metrics]({{< relref "#query-the-metrics-service" >}})** (or Metrics) is the platform service that provides a single source for monitoring Azure resources.
+- **[Azure Monitor Logs]({{< relref "#query-the-logs-service" >}})** (or Logs) gives you access to log data collected by Azure Monitor. -**[Azure Monitor Logs]({{< relref "#query-the-azure-resource-graph-service" >}})** (or Logs) gives you access to log data collected by Azure Monitor.
```

Comment:
It looks like you have this Azure Monitor Logs part twice with 2 different links is that intentional?

Reply:
Good catch!